code/game/gamemodes/steal_items.dm

```diff
@@ -14,7 +14,7 @@
 	var/list/areas = list()
 	var/flags=0
 
-/datum/theft_objective/proc/get_contents(var/obj/O)
+/datum/theft_objective/proc/get_contents(var/obj/O) //What a pile of shit. "What is this OOP you speak of?"
```

Comment:
To be fair we should begin allowing meme comments as long as it's not too outrageous. Just so people can have a laugh when they look at it.